client/core/core.go

```diff
@@ -2704,6 +2727,22 @@ func (c *Core) authDEX(dc *dexConnection) error {
 	if len(updatedAssets) > 0 {
 		c.updateBalances(updatedAssets)
 	}
+
+	// Order, match statuses should now be in sync with the DEX.
+	// Refresh the user's orders for each of the DEX's markets and
+	// log updated locked funds info.
+	dc.refreshMarkets()
+	dc.marketMtx.RLock()
+	for _, mkt := range dc.marketMap {
+		c.log.Infof("%s %s market - %d active orders, %s %s, %s %s locked for swapping, "+
+			"%s %s, %s %s locked in unspent contracts", dc.acct.host, mkt.Name, len(mkt.Orders),
+			amtString(mkt.BaseOrderLocked), mkt.BaseSymbol,
+			amtString(mkt.QuoteOrderLocked), mkt.QuoteSymbol,
+			amtString(mkt.BaseContractLocked), mkt.BaseSymbol,
+			amtString(mkt.QuoteContractLocked), mkt.QuoteSymbol)
```

Comment:
Just want to note that `amtString` is really just good for dividing by conversion factor since the string formatting cna be done with the printf specifiers, but also that `conversionFactor` is not 1e8 for all assets, and that's why I just log atoms or whatever the base unit is.  I get the desire to log in a more readable way though.
I'd prefer to remove this log loop entirely though once we're done using it for development.